eth/api.go

```diff
@@ -439,6 +440,19 @@ func (s *PrivateAccountAPI) NewAccount(password string) (common.Address, error)
 	return common.Address{}, err
 }
 
+func (s *PrivateAccountAPI) ImportAccount(privkey string, password string) (common.Address, error) {
+	hexkey, err := hex.DecodeString(privkey)
+	if err != nil {
+		return common.Address{}, err
+	}
```

Comment:
I think it would be nice to check the length of the private key here to make sure people don't send in some junk accidentally (i.e. shorter, longer).